hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenUpdateSpecs.java

```diff
@@ -386,7 +390,6 @@ public HapiSpec tooLongNameCheckHolds() {
                 .then(tokenUpdate("tbu").name(tooLongName).hasPrecheck(TOKEN_NAME_TOO_LONG));
     }
 
-    @HapiTest
```

Comment:
They are not passing, but had annotations. I've removed them so that we know that they need to be fixed.

Reply:
Right, this is how we agreed to handle this. The gradle task to run all e2e-tests should always pass. Currently, we do not run them in CI, but this will be changed soon.

If a test fails, it needs to be disabled until it is fixed.